projects/odf_transform/config.json

```diff
@@ -7,22 +7,45 @@
     "vocabularyFileList":[
 		"./vocabulary/all_vocabulary_list.json"
 	],
-    "organisationVocabulary": ["BIO","GF3"],
-    "summary": "brief summary of dataset",
-    "summary_fra": " summary in french here",
-    "title": "dataset title",
-    "title_fra": "title in french here",
-    "description": "detailed description",
-    "description_fra": "description in french",
-    "infoUrl": "http://www.pac.dfo-mpo.gc.ca/science/oceans/data-donnees/index-eng.html",
-    "keywords": "temperature, salinity",
-    "keywords_fra": "salinite",
-    "acknowledgement": "thanks etc ...",
-    "naming_authority": "CF Standard Name Table v29",
-    "creator_name": "CIOOS",
-    "creator_email": "cioos email here",
-    "creator_url": "http://www.someurlhere.com",
-    "license": "standard license information here",
-    "keywords_vocabulary": "GCMD Science Keywords",
-    "Conventions": "CF-1.6,CF-1.7,ACDD1.1"
+    "organisationVocabulary": ["IML","BIO","GF3"],
+    "global_attributes": {
+        "summary": "brief summary of dataset",
+        "summary_fra": " summary in french here",
+        "title": "dataset title",
+        "title_fra": "title in french here",
+        "description": "detailed description",
+        "description_fra": "description in french",
+        "infoUrl": "http://www.pac.dfo-mpo.gc.ca/science/oceans/data-donnees/index-eng.html",
+        "keywords": "temperature, salinity",
+        "keywords_fra": "salinite",
+        "acknowledgement": "thanks etc ...",
+        "naming_authority": "CF Standard Name Table v29",
+        "creator_name": "CIOOS",
+        "creator_email": "cioos email here",
+        "creator_url": "http://www.someurlhere.com",
+        "license": "standard license information here",
+        "keywords_vocabulary": "GCMD Science Keywords",
+        "Conventions": "CF-1.6,CF-1.7,ACDD1.1",
+        "naming_authority": "COARDS",
+        "comment": ""
+    },
+    "variables_from_header": {
```

Comment:
This looks like code more than a config setting, maybe should be in the code

Reply:
That is a way to list the different variables to be generated based on the available header metadata. I added a few features so that you can copy variables from other ones and potentially modify them with a apply or lambda function.